cmd/osbuild-service-maintenance/aws.go

```diff
@@ -13,10 +15,50 @@ import (
 	"github.com/osbuild/osbuild-composer/internal/cloud/awscloud"
 )
 
-func AWSCleanup(maxConcurrentRequests int, dryRun bool, accessKeyID, accessKey string, cutoff time.Time) error {
-	a, err := awscloud.New("us-east-1", accessKeyID, accessKey, "")
+type ChildToParentAssociation struct {
+	Child  string
+	Parent string
+}
+
+type MultiError struct {
```

Comment:
I think we want to use `errors.Join()` instead which gives us the stdlib "joinErrors" struct which does the same expect it also implements Unwrap (so that errors.{Is,As} keeps working)